README.md

```diff
@@ -17,72 +17,70 @@ If you have any issues with OBT, please check out [troubleshooting guide](https:
 
 2. Install the build tools globally:
 
-		npm install -g origami-build-tools
+	`npm install -g origami-build-tools`
 
 ## Usage
 
 	Usage
 		$ obt <command> [<options>]
 
 	Commands
-		build, b    Build CSS and JS in current directory
-		demo, d     Build demos into the demos directory
-		init        Initialise a new component with a boilerplate folder structure
-		install, i  Install npm and bower dependencies required to build modules
-		test, t     Run Origami specification tests and component specific tests
-		verify, v   Check folder and code structure follows Origami specification
+		develop, dev  Build demos locally every time a file changes and run a server to view them.
+		build, b      Build CSS and JS in current directory
+		demo, d       Build demos into the demos directory
+		init          Initialise a new component with a boilerplate folder structure
+		install, i    Install npm and bower dependencies required to build modules
+		test, t       Run Origami specification tests and component specific tests
+		verify, v     Check folder and code structure follows Origami specification
 
 	Options
 		-h, --help                 Print out this message
-		--watch                    Re-run every time a file changes
```

Comment:
I'd still want to have watch around for the `obt test` command

Reply:
hmm good point, I didn't think about that. Do you use `watch` with `obt test` whilst working on JS? Running the Sass tests and Pa11y on every change is not something I think is very valuable

Reply:
Yep, when working on the js tests

Reply:
Would `debug` work for your workflow? That's what I used to get immediate feedback on JS tests. I'm concerned `--watch` might be confusing as it sounds generic but isn't now

Reply:
hmm, yes it would. No need for --watch then :-)